extensions/resteasy-reactive/rest-client-reactive/deployment/src/test/java/io/quarkus/rest/client/reactive/StreamJsonTest.java

```diff
@@ -109,6 +114,21 @@ void shouldReadNdjsonFromSingleMessage() throws InterruptedException {
         assertThat(collected).hasSize(4).containsAll(expected);
     }
 
+    /**
+     * Reproduce <a href="https://github.com/quarkusio/quarkus/issues/30690">#30690</a>.
+     */
+    @Test
+    public void shouldReadUpToThreeTicks() {
+        createClient(uri)
+                .ticks()
+                .onItem()
+                .invoke(Objects::nonNull)
+                .subscribe()
+                .withSubscriber(AssertSubscriber.create(3))
+                // wait for 3 ticks plus some half tick ms of extra time (this should not be necessary, but CI is slow)
+                .awaitItems(3, Duration.ofMillis((TICK_EVERY_MS * 3) + (TICK_EVERY_MS / 2)));
```

Comment:
Should be duration of `(TICK_EVERY_MS * 2) ...` as 1st item is issued at time 0, 2nd at time 200ms, 3rd at time 400ms. So wait time 200ms * 2,5 should be correct

Reply:
This does not have to be precise taking into account that there are more things involved (REST call, subscriptions, etc). 
According to my experience, 200 ms of back time is enough to reproduce the issue and verify this is indeed fixed.